lib/weather_cast_angle/services/datetime_processor.ex

```diff
@@ -61,4 +61,14 @@ defmodule WeatherCastAngle.Services.DatetimeProcessor do
   """
   @spec is_current_date(String.t()) :: boolean()
   def is_current_date(date), do: date == get_current_date_string()
+
+  @doc """
+  Get yesterday "Asia/Tokyo" date from current, and return the "YYYY-MM-DD" formatted date string.
+  """
+  @spec get_yesterday_string_from_current() :: String.t()
+  def get_yesterday_string_from_current() do
```

Comment:
Although `get_yesterday_string_from_current` function was not used in this PR in the end, it was left in place and can be used in another implementation in the future.